Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -283,20 +287,6 @@ DasharoSystemFeaturesUiLibConstructor (
      */
     PRIVATE_DATA(SmallCoreActiveCount) = DASHARO_CPU_CORES_ENABLE_ALL;
     PRIVATE_DATA(BigCoreActiveCount) = DASHARO_CPU_CORES_ENABLE_ALL;
-    gRT->SetVariable (

```

Comment:
Why this disappeared? This isn't mentioned in the commit messages.

Reply:
It is a constructor which should initialize the form data based on variables, not create them. Thus it is removed.

Reply:
Reverted it back and added a PCD IF

Reply:
It is, but only when someone enters the form and saves changes. That is why it was added earlier.

Reply:
So, does that mean that all of the rest of variables isn't created until user enters setup? That would mess up the measurements...

Reply:
See https://github.com/Dasharo/DasharoModulePkg/pull/45 for more context

Reply:
> See #45 for more context

I've seen it, and what I see there:

> - Value for variable initialization and resetting now needs to be specified only in one place, getting rid of bugs when they diverge (see below).
> - Duplicated code was replaced with macros: adding a new variable now requires adding 1-2 new lines in several places.

This isn't in line with what happens here. Variable are set in multiple places, and one of those doesn't use macros.

Asking as simply as i can: Why is this treated differently than any other variable? How can these values be set to invalid values, and why other settings can't?

I'm expecting one of these outcomes:
- All variables are validated here.
- No variables are validated here.
- Just variables with core settings are validated, **with clearly mentioned reason why** in the comment.

Reply:
My mistake, there is a comment in the non-modified portion that I missed.